plugins/dell/fu-self-test.c

```diff
@@ -114,7 +114,8 @@ fu_plugin_dell_tpm_func (void)
 	g_assert (device_alt != NULL);
 
 	/* make sure not allowed to flash 1.2 */
-	g_assert_false (fu_device_has_flag (device_alt, FWUPD_DEVICE_FLAG_UPDATABLE));
+	g_assert_false (
```

Comment:
I think it's okay to relax the line lengths in some files, e.g. the tests. Not sure how possible that is to achieve.

Reply:
Not sure either on this.  Might be able to detect "test" in the file name and relax this particular style rule when clang is called on it.